internal/rollout/rollout_test.go

```diff
@@ -293,7 +305,15 @@ func TestUpdateService(t *testing.T) {
 			lastReady: "test-002",
 			healthCriteria: []config.HealthCriterion{
 				{Metric: config.RequestCountMetricsCheck, Threshold: 1500},
-				{Metric: config.ErrorRateMetricsCheck, Threshold: 0.95},
+				{Metric: config.ErrorRateMetricsCheck, Threshold: 5},
+			},
+			outAnnotations: map[string]string{
+				rollout.StableRevisionAnnotation:    "test-001",
+				rollout.CandidateRevisionAnnotation: "test-002",
+				rollout.LastHealthReportAnnotation: "status: inconclusive\n" +
```

Comment:
potentially it's gonna be a hassle to provide full values here.

Reply:
Yeah, let's work on this in another PR